pennylane/workflow/execution.py

```diff
@@ -421,10 +476,10 @@ def execute(
     cache: Union[None, bool, dict, Cache] = True,
     cachesize=10000,
     max_diff=1,
-    override_shots: int = False,
-    expand_fn="device",  # type: ignore
-    max_expansion=10,
-    device_batch_transform=True,
+    override_shots: int = UNSET,
+    expand_fn=UNSET,  # type: ignore
+    max_expansion=None,
+    device_batch_transform=None,

```

Comment:
Why not use `UNSET` as the default for these two as well?

Reply:
I could have, but I reserved using UNSET for the cases where None was a valid value. For `max_expansion` and `device_batch_transform` that does not seem to be the case

Reply:
Why do we need to use `UNSET` here? E.g., for `override_shots`, could we not just keep the value to `False` and only raise an error if the default is not used?

Reply:
The idea is that I wanted the warning to be raised every time the argument was used. This includes the case when argument was called explicitly with the default value,  e.g. `override_shots=False`. So I changed all the arguments to a default value that would be "invalid" (either None or UNSET) which would provide me with this information, and then set it back to the original default value.